lesson7Homework/lesson7Homework/Vector.cpp

```diff
@@ -0,0 +1,179 @@
+#include "Vector.h"
+#include<iostream>
+#include <ctime>
+
+Vector::Vector(int size)
+{
+	size_ = size;
+	vectorOfData_ = new int[size];
+    for (int i{}; i < size_; i++)
+        *(vectorOfData_+i) = 0;
```

Comment:
Достаточно так, все элементы про инициализируются пустыми значениями.
```suggestion
	vectorOfData_ = new int[size]{};
```